### DISCUSSION TITLE
Incrementing variables by 1, ++, --

### DISCUSSION THREAD

**Discussion by enenbee** [godotengine/godot-proposals#4547] [Scripting]
I don't know if i'm adding this suggestion correctly, but it would be useful if in GDScript, like other programming languages, there could be a shorthand for incrementing / decrementing a variable by 1

**Comment by Calinou** [comment]
This is already being tracked in https://github.com/godotengine/godot-proposals/issues/2869. That said, this was already requested in https://github.com/godotengine/godot/issues/1107 and it was rejected.
